test/cli_test.rb

```diff
@@ -3,26 +3,27 @@
 require 'fileutils'
 
 class CliTest < Test::Unit::TestCase
+  HOME_DIR = File.expand_path(File.dirname(__FILE__))
+  TEST_HOME = File.join(HOME_DIR, 'files', 'home')
+  CONFIG_DIR = File.join(TEST_HOME, '.shopify', 'shops')
+  DEFAULT_SYMLINK = File.join(CONFIG_DIR, 'default')
+
   def setup
-    @test_home = File.join(File.expand_path(File.dirname(__FILE__)), 'files', 'home')
-    @shop_config_dir = File.join(@test_home, '.shopify', 'shops')
-    @default_symlink = File.join(@shop_config_dir, 'default')
-    `rm -rf #{@test_home}`
-    ENV['HOME'] = @test_home
+    `rm -rf #{TEST_HOME}`
```

Comment:
Can we wrap this and the following `rm -rf`'s in a helper method which accepts a path param. It makes me anxious seeing it in several places.